docs/projects/ad7616_sdz/index.rst

```diff
@@ -0,0 +1,336 @@
+.. _ad7616_sdz:
+
+AD7616-SDZ HDL project
+===============================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The :part:`AD7616 <AD7616>` is a 16-bit, data acquisition system (DAS)
```

Comment:
In all the places where you used the `:part:` role, you can drop the text outside the `< >` because it will automatically render to the name of the part

Reply:
Done.